flow/layers/container_layer.cc

```diff
@@ -86,8 +88,9 @@ void ContainerLayer::PaintChildren(PaintContext& context) const {
 void ContainerLayer::TryToPrepareRasterCache(PrerollContext* context,
                                              Layer* layer,
                                              const SkMatrix& matrix) {
-  if (!context->has_platform_view && context->raster_cache &&
-      SkRect::Intersects(context->cull_rect, layer->paint_bounds())) {
+  if (!context->has_platform_view && context->raster_cache
+  //  &&    SkRect::Intersects(context->cull_rect, layer->paint_bounds())
```

Comment:
I commented this out until I prove to myself that it isn't going to cause a similar problem to the issue being considered.

Reply:
I reverted this change. The net effect if the intersection fails will be to avoid preparing the layer cache for the layer. Since a failure of this test would mean that this branch will not be rendered, this should not affect rendering and if there is an error in the logic, it just means that the rendering would be done on the fly - the results would still be correct.